_overviews/contributors/index.md

```diff
@@ -285,72 +248,62 @@ uid                  Julien Richard-Foy <julien.richard-foy@epfl.ch>
 
 In my case, I have one key pair, whose ID is `BE614499`.
 
-Export your public and private keys into files, in a `ci` directory:
-
-~~~ bash
-$ mkdir ci
-$ gpg -a --export (key ID) > ci/pubring.asc
-$ gpg -a --export-secret-keys (key ID) > ci/secring.asc
-~~~
 
+Then:
+
+ 1. Create a new Secret containing the passphrase of your PGP key named `PGP_PASSPHRASE`.
+ 2. Create a new Secret containing the base64 encoded secret of your private key which you can obtain
+ by running: 
+```
+# macOS
+gpg --armor --export-secret-keys $LONG_ID | base64
+# Ubuntu (assuming GNU base64)
+gpg --armor --export-secret-keys $LONG_ID | base64 -w0
+# Arch
+gpg --armor --export-secret-keys $LONG_ID | base64 | sed -z 's;\n;;g'
+# FreeBSD (assuming BSD base64)
+gpg --armor --export-secret-keys $LONG_ID | base64
+# Windows
+gpg --armor --export-secret-keys %LONG_ID% | openssl base64
+```
+ 3. Publish your public key signature to a public server, for example [http://keyserver.ubuntu.com:11371](http://keyserver.ubuntu.com:11371/).
+ You can obtain the signature by running:
```

Comment:
This step was not mentioned before your changes, is it necessary/useful?

Reply:
https://central.sonatype.org/publish/requirements/gpg/ says that the public key must be distributed to a public server so that users can verify artifacts